tls_test.go

```diff
@@ -43,7 +43,11 @@ func TestTLS(t *testing.T) {
 }
 
 func createPrivKey(priv *rsa.PrivateKey) string {
-	f, _ := ioutil.TempFile(".", "priv*.key")
+	f, err := os.CreateTemp(".", "priv*.key")
+	if err != nil {
+		panic(err)
+	}
+
```

Comment:
```suggestion
	require.NoError(t, err)

```

Reply:
Then `t` needs to be passed. Which is of course possible.

Reply:
Oh sorry, missed that. Yeah I think that makes sense since it's in the same file and not an external test helper library. Ignore if you will, PR approved :)